lib/index.js

```diff
@@ -9,12 +9,8 @@
  *   being needed.
  *   Copied from: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a2bc1d8/types/node/globals.d.ts#L174>
  *
- * @typedef {string|Uint8Array} VFileValue Contents of the file.
- *   Can either be text, or a Buffer like structure.
- *   This does not directly use type `Buffer`, because it can also be used in a
- *   browser context.
- *   Instead this leverages `Uint8Array` which is the base type for `Buffer`,
- *   and a native JavaScript construct.
+ * @typedef {typeof globalThis extends {Buffer: {from: (string: any) => infer T}} ? T : never} Buffer
```

Comment:
Is the conditional needed?
When I inspect the generated typing file, it gets compiled down to
```ts
export type Buffer = globalThis.Buffer
```
Maybe `globalThis.buffer` could be used directly?

Reply:
is this mergable as-is or should we wait?

Reply:
It doesn’t work as-is. I created sindresorhus/type-fest#249.

If that isn’t accepted, this type needs to be defined in a `.d.ts` file instead.

Reply:
Would that even work though, due to:

> Note that this needs to be in every file in which URL is used, as if it was declared global, it would cause a circular type definition. So it's a workaround, not a solution.

(from https://github.com/DefinitelyTyped/DefinitelyTyped/issues/34960#issuecomment-576906058)

Reply:
This doesn’t declare a global `Buffer`. It defines a local and exported `Buffer` type in this file.